src/common/utils/format.ts

```diff
@@ -71,3 +71,27 @@ export const formatToTitleCase = (string: string): string =>
   )
 
 export const formatDate = (time: Dayjs): string => time.format('MM/DD/YY')
+
+export const formatFileSize = (size: number): string => {
+  let i = -1
+  const units = [' kB', ' MB', ' GB', ' TB', 'PB', 'EB', 'ZB', 'YB']
+  do {
+    size = size / 1024
+    i++
+  } while (size > 1024)
+
+  if (!i) {
+    return Math.round(size) + units[i]
+  }
+
+  return Math.max(size, 0.1).toFixed(1) + units[i]
+}
+
+const numFormatter = new Intl.NumberFormat('en-US', {
+  style: 'decimal',
+  maximumFractionDigits: 0,
+})
+
+export const formatPSL = (val: number): string => {
+  return numFormatter.format(val) + ' PSL'
```

Comment:
Removed formatPSL func and changed related components to use one way with other components (formatNumber + currencyName in component).
But situation with currency name is a little bit strange. Many components use local consts, wallet screen use fetchInfo() from src/api/pastel-rpc/control.ts. Some components pass currency name to subcomponents.
If project is single-currency, it would be better to set currency name in one place (global state or something like this) and use from that place. Also single util function to format value to currency string will be very useful.